src/methods/send.ts

```diff
@@ -93,4 +88,52 @@ const send = async (
   }
 };
 
+const setFocusObject = (action: BrowserSetFocusAction, genomeBrowser: GenomeBrowserType) => {
+  const { focusType } = action.payload;
+
+  if (focusType === 'gene') {
+    setFocusGene(action, genomeBrowser);
+  } else if (focusType === 'location') {
+    setFocusLocation(action, genomeBrowser);
+  }
```

Comment:
Won't this be a candidate for `switch`?

Reply:
I dunno. The `send` method above works entirely on if-elses. I'm a bit scared of javascript switches, due to their fall-through behaviour, and how easy it is to forget to `break`.